includes/classes/PPMFWC/Gateways.php

```diff
@@ -375,7 +375,13 @@ public static function getOrderReturnUrl(WC_Order $order, $newStatus)
             $url = add_query_arg('paynl_status', PPMFWC_Gateways::STATUS_CANCELED, wc_get_checkout_url());
         } elseif ($newStatus == PPMFWC_Gateways::STATUS_DENIED)
         {
-            wc_add_notice(esc_html(__('Payment denied. Please try again or use another payment method.', PPMFWC_WOOCOMMERCE_TEXTDOMAIN)), 'error');
+            $methodName = $order->get_payment_method_title();
+            if (!empty($methodName)) {
+                wc_add_notice(esc_html(sprintf(__('Unfortunately the payment has been denied by %s. Please try again or use another payment method.', PPMFWC_WOOCOMMERCE_TEXTDOMAIN), $methodName)), 'error');
+            } else {
+                wc_add_notice(esc_html(sprintf(__('Unfortunately the payment has been denied by the payment method. Please try again or use another payment method.', PPMFWC_WOOCOMMERCE_TEXTDOMAIN), $methodName)), 'error');
```

Comment:
als methodName empty is hoeft hij hier tog niet meer in de functie meegegeven te worden?

Reply:
![Screenshot from 2021-07-29 08-43-12](https://user-images.githubusercontent.com/61683999/127444495-340feded-812f-4669-bfd3-493b49d74bfd.png)

Lijkt er op dat de vertaling niet goed gaat?
Ik zie deze tekst ook niet terugkomen in de .po of in de .mo van normaal nederland en formeel.

Reply:
![Screenshot from 2021-07-29 08-51-49](https://user-images.githubusercontent.com/61683999/127445517-7c825adb-d1a9-4f29-a48f-76f827433c12.png)

Nog een vertaling die mist